crates/rethnet_eth/src/remote/cacheable_method_invocation.rs

```diff
@@ -91,6 +91,7 @@ impl<'a> CacheableMethodInvocation<'a> {
         Some(ReadCacheKey(cache_key))
     }
 
+    #[allow(clippy::match_same_arms)]
```

Comment:
Why not combine them?

Reply:
Ah, thought this was resolved:, but I see now that it wasn't: https://github.com/NomicFoundation/hardhat/pull/4319/files#r1310095396

Reply:
Sorry, I didn't see that comment